assignments/objects.js

```diff
@@ -20,25 +20,72 @@ const example = {
 // Write your intern objects here:
 
 
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi, F",
+  "email": "mmelloy0@psu.edu", 
+  "gender": "F",
+}
+
+const intern2 = {
+  "id": 2,
+  "name": "Kennan,M", 
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function() {
+    return "Hello my name is Kennan";
+  }
+}
+
+const intern3 = {
+  "id": 3,
+  "name": "Keven,M",
+  "email": "mmelloy0@kmummery2@wikimedia.org", 
+  "gender": "M",
+}
+
+const intern4 = {
+  "id": 4,
+  "name": "Gannie,M",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "F",
+}
+
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta,F",
+  "email": "adaine5@samsung.com", 
+  "gender": "F",
+  multiplyNums: function(num1, num2) {
+    return "I can multiply numbers  " + num1 + num2;
+  }
+}
+
+
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(Object.entries(intern1)[1]);
```

Comment:
I understand the idea behind using `Object.entries` and it was nice of you to log both the object property and value.

Of course, you can't always rely on the index from `Object.entries` because you will encounter objects that may have a lot of properties (to the point where it wouldn't be practical to track the index of a property). In the future, it will usually be better for you to use dot notation (`objName.keyName`) or bracket notation  (`objName["keyName"]`).